fulcio.proto

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.fulcio.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/sigstore/fulcio/pkg/generated/protobuf";
+
+service CA {
+    /**
+     * Returns information about the current state of the transparency log
+     */
+    rpc GetSigningCert(CertificateRequest) returns (CertificateResponse){
+        option (google.api.http) = {
+          post: "/api/v1/signingCert"
+          body: "*"
+        };
+    }
+    /**
+     * Returns the public key that can be used to validate the signed tree head
+     */
+    rpc GetRootCertificate(google.protobuf.Empty) returns (RootCertificateResponse){
+        option (google.api.http) = {
+          get: "/api/v1/rootCert"
+        };
+    }
+    /**
+     * Returns the current version information for this Fulcio instance
+     */
+    rpc Version(google.protobuf.Empty) returns (VersionResponse){
+        option (google.api.http) = {
+          get: "/api/v1/version"
+        };
+    }
+}
+
```

Comment:
There's some stylistic changes I'd recommend - https://google.aip.dev/general describes a style for APIs, and if we follow it closely, the GRPC tooling should work nicely.